packages/gatsby-transformer-pdf/src/gatsby-node.js

```diff
@@ -41,6 +41,7 @@ async function onCreateNode({
 
   pdfNode.content = parsedContent
   pdfNode.internal.contentDigest = createContentDigest(pdfNode)
+  pdfNode.relativePath = node.relativePath
```

Comment:
That's my bad for asking for this! As @KyleAMathews said, we can probably rely on the relativePath query with the _File_ node rather than adding anything specific to this plugin. Maybe remove?

Reply:
Ok. i removed it.  can you help me with the unit test?